app/Policies/AchievementCommentPolicy.php

```diff
@@ -68,9 +72,12 @@ public function update(User $user, AchievementComment $comment): bool
 
     public function delete(User $user, AchievementComment $comment): bool
     {
+        if ($comment->is_automated) {
+            return false;
+        }
+
```

Comment:
I'm unable to delete my own comments without the MODERATOR role.

Reply:
Great catch. Looks like there was a bug in the policy. Fixed in latest.